package.json

```diff
@@ -89,6 +89,7 @@
     "@typescript-eslint/parser": "^4.31.2",
     "axios": "^0.23.0",
     "classnames": "^2.3.1",
+    "dayjs": "^1.11.3",
```

Comment:
*sad momentjs noises in distance*

Reply:
yeah... sure 🤡
![image](https://user-images.githubusercontent.com/32616724/178343749-5dc8860a-2089-4fd2-b0b1-abeb77410cdb.png)